### DISCUSSION TITLE
Problem with working on step - invert_network

### DISCUSSION THREAD

**Discussion by chitrabuth** [insarlab/MintPy#1193] [Q&A]
I am using Mintpy to generate deformation velocity map using smallbaselineApp.py.
Now I am working on step - invert_network. However, an error occurred. I have no idea how to fix it but I hope you could help by checking the error message attached.
[error message.txt](https://github.com/insarlab/MintPy/files/15282331/error.message.txt)

**Comment by welcome** [comment]
👋  Thanks for opening your first issue here! Please filled out the template with as much details as possible. We appreciate that you took the time to contribute!
Make sure you read our [contributing guidelines](https://github.com/insarlab/MintPy/blob/master/docs/CONTRIBUTING.md).

**Comment by yunjunz** [comment]
I would recommend to turn off the dask parallelization, then check the related error message for a more meaningful debugging.
